src/API/Google/AdsCampaign.php

```diff
@@ -99,15 +99,17 @@ public function __construct( GoogleAdsClient $client, AdsCampaignBudget $budget,
 	/**
 	 * Returns a list of campaigns with targeted locations retrieved from campaign criterion.
 	 *
-	 * @param bool $exclude_removed Exclude removed campaigns (default true).
-	 * @param bool $fetch_criterion Combine the campaign data with criterion data (default true).
+	 * @param bool  $exclude_removed Exclude removed campaigns (default true).
+	 * @param bool  $fetch_criterion Combine the campaign data with criterion data (default true).
+	 * @param array $args Arguments for the Ads Campaign Query for example: per_page for limiting the number of results.
+	 * @param bool  $return_pagination_params Whether to return pagination params (default false).
 	 *
 	 * @return array
 	 * @throws ExceptionWithResponseData When an ApiException is caught.
 	 */
-	public function get_campaigns( bool $exclude_removed = true, bool $fetch_criterion = true ): array {
+	public function get_campaigns( bool $exclude_removed = true, bool $fetch_criterion = true, $args = [], $return_pagination_params = false ): array {
```

Comment:
What's the value of NOT returning pagination params? Even if they are not needed I assume they are not harmful. So maybe we can simplify this method by returning them always.

Reply:
@puntope, the reason for not always returning the pagination params is that we would need to refactor other parts of the code and tests that depend on this method:

- https://github.com/woocommerce/google-listings-and-ads/blob/6ff59162df0fc9c79e1f0c1af8267a764fe143a9/src/API/Google/AdsCampaign.php#L405
- https://github.com/woocommerce/google-listings-and-ads/blob/6ff59162df0fc9c79e1f0c1af8267a764fe143a9/src/MultichannelMarketing/GLAChannel.php#L210
- https://github.com/woocommerce/google-listings-and-ads/blob/6ff59162df0fc9c79e1f0c1af8267a764fe143a9/tests/Unit/API/Google/AdsCampaignTest.php#L95
- ..etc

Do you think it’s worth refactoring those parts even if they don't require pagination?